composer.json

```diff
@@ -43,17 +43,20 @@
 		}
 	},
 	"scripts": {
+		"ci": [
+			"composer test",
+			"composer cs",
+			"composer md"
+		],
 		"test": [
 			"composer validate --no-interaction",
 			"phpunit"
 		],
 		"cs": [
```

Comment:
Which raises the question why we need two tools to do the same? Or the other way around, if they are not the same, why shouldn't it be possible to run them independently?